src/Model/GroundWaterFlow/gwf3.f90

```diff
@@ -309,8 +309,8 @@ subroutine gwf_ar(this)
     class(BndType), pointer :: packobj
 ! ------------------------------------------------------------------------------
     !
-    ! -- Allocate and read modules attached to model
-    if (this%inic > 0) call this%ic%ic_ar(this%x)
+    ! -- Load modules attached to model
+    if (this%inic > 0) call this%ic%ic_load(this%x)
```

Comment:
Unless there is some reason we need to wait until AR, I think we should call this ic_load method as soon as possible, maybe toward the end of ic_create?

Reply:
After trying this out, it seems this needs to come after define (`*_df`), I think because that's where discretization sources its info, so I left it in AR for now.